README.md

```diff
@@ -28,20 +29,20 @@ Here's the implementation file. This view's height can change based on the Dynam
 - (void)sharedInit {
     [super sharedInit];
     // Create this view's subviews
-    UIImageView *imageView = [[UIImageView alloc] initWithImage:[UIImage imageNamed:@"MyImage.png"]];
+    _imageView = [[UIImageView alloc] initWithImage:[UIImage imageNamed:@"MyImage.png"]];
     [self addSubview:imageView];
```

Comment:
Yeah I made UIImageView a instance variable to highlight the user of view in the the layout block... view.imageView. (So you don't have to do a weak self reference.)

So for example later you can access the imageView to update it in an instance method (like setting an image URL).

This might be confusing though.